### DISCUSSION TITLE
Incomplete Absorption

### DISCUSSION THREAD

**Discussion by ranamjz** [Open-Systems-Pharmacology/Forum#1084] [Questions & Problems]
Hello everyone i have been trying to perform a simulation for Rivaroxaban , This medication is not administered intravenously so its 10 mg dose usually has a compelete absorption . But when i get the results of my simulation it always gives me this error : ⚠️ absorption seems to be incomplete and pk parameters should be compared with an iv simulation!
[20220116_134959.jpg](https://user-images.githubusercontent.com/90548695/149655955-d04b31a8-08bf-4da3-b0e4-1eec6ae51dcc.jpg)

What should i do
Your help is much appreciated

**Comment by StephanSchaller** [comment]
Hi @ranamjz , your simulation time should be extended.

After 20h, drug is likely still transiting the GI tract and thus the PK-Parameters might not yet have the "correct" (or final) value.

**Reply by ranamjz** [reply]
Dear mr Schaller 
Thank you for replying but the trouble is that the medication is given every 24hrs orally and i have ran the simulation for 4 weeks so the duration is long enough

**Reply by msevestre** [reply]
From your graph, it looks more like 30 hrs
![image](https://user-images.githubusercontent.com/1041237/150321894-d0d88358-986a-4688-860c-7657795b0b64.png)
CAn you take a proper screenshot  or copy the graph to image using right click

**Reply by ranamjz** [reply]
Of course and thank you very much 
![20220124_112205.jpg](https://user-images.githubusercontent.com/90548695/150742771-278ca6b4-09cc-4635-bda8-171c2ec34d85.jpg)

**Reply by ranamjz** [reply]
![20220124_113309.jpg](https://user-images.githubusercontent.com/90548695/150744172-9a936366-22e3-43b8-ac36-f9b1eacfc0f7.jpg)

**Comment by arketkar** [comment]
@ranamjz 
You may try running the simulation for a longer time. Like 72 hours or more.
Regards.
Anant

**Comment by jhunt8** [comment]
Thanks for this, all! To corroborate, I observe the same warning after 120h simulation and fraction absorbed 0.96. For additional information: formulation building block "use as suspension" = yes.

It is useful to trigger that warning if GI transit is incomplete, but I wonder if something else is triggering in this case.

![Incomplete_Absorption2](https://user-images.githubusercontent.com/87446546/160673786-14c5833e-b5b8-4d89-9836-26c5bdf42835.PNG)

**Reply by ranamjz** [reply]
Thank you . 
My medication is used as a tablet and not a suspension and it is normal not to have a FA of 1 in orally administered medications . I could fix FA after i changed the intestinal permeability (transcellular) in my simulation . Also you can check other values such as VSS to see if you are getting realistic results after .😊
